src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/UpdateAlertIntTest.kt

```diff
@@ -0,0 +1,241 @@
+package uk.gov.justice.digital.hmpps.hmppsalertsapi.resource
+
+import org.assertj.core.api.Assertions
```

Comment:
Really minor: IntelliJ puts this in as `import org.assertj.core.api.Assertions` rather than `import org.assertj.core.api.Assertions.assertThat` when copying meaning that all the `assertThat`s become `Assertions.assertThat`. You could change the import to remove that `Assertions.` everywhere

Reply:
Ah yes - Done.